cpp-client/build-dependencies.sh

```diff
@@ -5,12 +5,30 @@
 #
 
 #
-# Tested on Ubuntu 20.04
+# Tested on Ubuntu 22.04 and Fedora 28.
```

Comment:
I don't think it's good practice to release new code that is tested on "Fedora 28". If we are really targetted RedHat or Centos, we should really say / test for that.

Reply:
See above, we (I) was testing on Fedora 28, which is exactly what the comment says.

Reply:
So; I don't think we should release new code that claims to work on "Fedora 28" - it's an anti-feature IMO. Fedora 28 has been EOL since 2019. No development work whatsoever has happened since then.

If we are looking for RHEL 8 compatibily, we should explicitly call out _that_ compatibility (and/or Centos 8) instead of Fedora 28.

Reply:
Ok, I don't agree but I don't care enough.  I removed the comment, we don't claim to work on fedora 28 anymore.